README.md

```diff
@@ -48,9 +49,8 @@ Fields:
 ## Known limitations
 
 There are two big limitations that prevent this plugin from being useful for production:
-
-1. Until RethinkDB supports `return_initial` on all changefeeds ([which it should in 2.1](https://github.com/rethinkdb/rethinkdb/issues/3579)), this plugin won't do "backfilling" of documents already in a table. It only sends changes to the table as they are received. When `return_initial` is implemented fully, this plugin will be updated to take advantage of that capability to provide backfilling.
-2. Until RethinkDB supports resuming changefeeds ([which it should in 2.1](https://github.com/rethinkdb/rethinkdb/issues/3471)), this plugin cannot guarantee that no changes are missed if a connection to the database is dropped. Again, once that functionality is implemented, this plugin will be modified to provide reliable "at least once" semantics for changes (meaning once it reconnects, it can catch back up and send any changes that it missed).
+1. Until RethinkDB supports resuming changefeeds ([which it should in 2.3](https://github.com/rethinkdb/rethinkdb/issues/3471)), this plugin cannot guarantee that no changes are missed if a connection to the database is dropped. Again, once that functionality is implemented, this plugin will be modified to provide reliable "at least once" semantics for changes (meaning once it reconnects, it can catch back up and send any changes that it missed).
```

Comment:
I think we should remove the reference to version 2.3, since there are different degrees of resumable changefeeds. The version that might make it into 2.3 will still not be able to recover if the client or server are shut down (or fail) for example.